### PR TITLE
Update cheatsheet.md

### PR DESCRIPTION
**Pull request by Ashutosh257**
Updated Media Controller API for chat:

 WE HAVE .getConnections() [Plural] and not .getConnection() [Singular]
 It returns an array with connection between the AGENT and CUSTOMER 

connection.getMediaType() === chat , AND  !== connect.MediaType.CHAT
connection.getMediaType() returns "chat" and not connect.MediaType.CHAT

there is no such function as "onDisconnect()" in connect-streams.js

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: cheat-sheet.md [hunk 1]
```diff
@@ -233,28 +233,51 @@ Download agent logs
...
+	var connectionId = contact.getAgentConnection()['connectionId']
+	
+	// WE HAVE .getConnections() [Plural] and not .getConnection() [Singular]
+        //It returns an array with connection between the AGENT and CUSTOMER 
+	
+	contact.getConnections().forEach(function (connection) {
+	//console.log(connection)
+	
+	//connection.getMediaType() === chat , AND  !== connect.MediaType.CHAT
+        // connection.getMediaType() returns "chat" and not connect.MediaType.CHAT
```
Comment (ctwomblyamzn) [inline cheat-sheet.md:248]: connect.MediaType.CHAT is defined as "chat"

File: cheat-sheet.md [hunk 2]
```diff
@@ -233,28 +233,51 @@ Download agent logs
...
+						}
+					})
+
+					controller.sendMessage({ message: "hey ", contentType: "text/plain" })
+						.then(function (res, req) {
+							console.log(res.httpResponse.statusCode)
+							//console.log(res.status)
+					});
+
+					// there is no such function as "onDisconnect()" in connect-streams.js
```
Comment (ctwomblyamzn) [inline cheat-sheet.md:267]: Yes, you are right. This code snippet relies on Chat functionality, meaning that ChatJS should be used as well. There are notes in Documentation.md and README.md about this. `controller.onDisconnect()` comes from ChatJS's `ChatController`.

File: cheat-sheet.md (hunk 1 above)
Comment (ctwomblyamzn) [reply cheat-sheet.md:248]: See [here](https://github.com/amazon-connect/amazon-connect-streams/blob/f046b812073daadecc9370ae4649e95990d818e3/src/api.js#L136)

File: cheat-sheet.md (hunk 2 above)
Comment (ctwomblyamzn) [reply cheat-sheet.md:267]: Can you change this to be more clear about where onDisconnect comes from?

Comment (Ashutosh257) [reply cheat-sheet.md:267]: yes sure

**Pull request closed without merging**